apollo-compiler/src/main/kotlin/com/apollostack/compiler/ir/GraphQlType.kt

```diff
@@ -1,11 +1,13 @@
 package com.apollostack.compiler.ir
 
+import com.apollostack.compiler.normalizeTypeName
+import com.squareup.javapoet.AnnotationSpec
 import com.squareup.javapoet.ClassName
 import com.squareup.javapoet.ParameterizedTypeName
 import com.squareup.javapoet.TypeName
+import javax.annotation.Nullable
```

Comment:
Either way works fine. I guess android support annotations is better. I'll change it